docker-compose.yml

```diff
@@ -15,189 +18,287 @@ services:
+      - UnitCellDatabaseConfiguration__Databases__0__WriteConnectionString=${DB_CONNECTIONSTRING};SearchPath=unitcell;Database=${CSD_DATABASE}
+      - Logging__LogLevel__Default=Warning
+    logging:
+      driver: local
 
   ccdc-csd-similaritysearch:
     labels:
       <<: *k8s-labels
     depends_on:
-      - ccdc-csd-fingerprint
+      ccdc-csd-fingerprint:
+        condition: service_healthy
     image: ccdcrepository.azurecr.io/onsite/ccdc-csd-structure-similarity-search:4.0.0
     environment:
       - StructureSimilaritySearchReadConnection=${DB_CONNECTIONSTRING};SearchPath=similarity;Database=${CSD_DATABASE}
+      - Logging__LogLevel__Default=Warning
     restart: unless-stopped
+    logging:
+      driver: local
 
   ccdc-csd-fingerprint:
     labels:
       <<: *k8s-labels
-    image: ccdcrepository.azurecr.io/onsite/ccdc-csd-fingerprint-service:4.0.0
+    image: ccdcrepository.azurecr.io/onsite/ccdc-csd-fingerprint-service:4.2.0
     restart: unless-stopped
+    logging:
+      driver: local
 
   ccdc-csd-substructure-filter:
     labels:
       <<: *k8s-labels
     image: ccdcrepository.azurecr.io/onsite/ccdc-csd-substructure-filter-service:4.0.0
     restart: unless-stopped
+    logging:
+      driver: local
 
   ccdc-csd-crystal-structure-export:
     labels:
       <<: *k8s-labels
     image: ccdcrepository.azurecr.io/onsite/ccdc-csd-crystal-structure-export-service:4.1.0
     restart: unless-stopped
+    logging:
+      driver: local
 
   ccdc-csd-reducedcell-calculation-service:
     labels:
       <<: *k8s-labels
     image: ccdcrepository.azurecr.io/onsite/ccdc-csd-reducedcell-calculation-service:4.0.0
     restart: unless-stopped
+    logging:
+      driver: local
 
   webcsd-theory:
     labels:
       <<: *k8s-labels
     image: ccdcrepository.azurecr.io/onsite/csd-theory:4.0.0
     environment:
       - CCDC_LICENSING_CONFIGURATION=${CCDC_LICENSING_CONFIGURATION}
+      - Logging__LogLevel__Default=Warning
     restart: unless-stopped
+    logging:
+      driver: local
 
   message-broker:
-      image: rabbitmq:3
-      environment:
-        RABBITMQ_DEFAULT_USER: queueuser
-        RABBITMQ_DEFAULT_PASS: ${CCDC_LICENSING_CONFIGURATION}
-      restart: unless-stopped 
+    image: rabbitmq:3
+    environment:
+      RABBITMQ_DEFAULT_USER: queueuser
+      RABBITMQ_DEFAULT_PASS: ${CCDC_LICENSING_CONFIGURATION}
+    restart: unless-stopped
+    healthcheck:
+      test: ["CMD", "rabbitmqctl", "await_startup"]
+      interval: 30s
+      timeout: 10s
+      retries: 5
+      start_period: 30s
+    logging:
+      driver: local
 
   ccdc-file-generation:
     restart: unless-stopped
     image: ccdcrepository.azurecr.io/onsite/ccdc-structure-file-generation:4.1.0
     environment:
-      - CacheSettings__ConnectionString=${DB_CONNECTIONSTRING};Database=structure_file_cache
+      - CacheSettings__ConnectionString=${DB_CONNECTIONSTRING};Database=testsuite_structure_file_cache
```

Comment:
Just checking this should be test suite_*

Reply:
Good spot!